docs/serving/samples/tag-header-based-routing/README.md

```diff
@@ -0,0 +1,118 @@
+This sample illustrates the feature "Tag Header Based Routing".
+
+## Tag Header Based Routing Feature
+
+The feature "Tag Header Based Routing" allows users to send requests directly to specific tagged Revisions with
+the same URL of Knative Service. In order to achieve this, users only need to set the specific header "Knative-Serving-Tag:
+{revision-tag}" into the request.
+
+Currently Istio and Contour Ingress support this feature.
```

Comment:
Could you please add Kourier in the list? It was implemented and landed in v0.16 release https://github.com/knative/net-kourier/pull/110

Reply:
Done